pkg/fuse/fuse.go

```diff
@@ -347,6 +347,9 @@ func (fs *fileSystem) OpenDir(cancel <-chan struct{}, in *fuse.OpenIn, out *fuse
 	defer releaseContext(ctx)
 	fh, err := fs.v.Opendir(ctx, Ino(in.NodeId), in.Flags)
 	out.Fh = fh
+	if fs.conf.ReaddirCache {
```

Comment:
We should check if the directory is changed or not, similar to files

Reply:
we should also check the version of kernel module,  require 4.20+

Reply:
How to track if the dir has changed or not, using mtime? kernel already did the [check](https://lore.kernel.org/lkml/20180928154234.19270-9-mszeredi@redhat.com/)